v2/pkg/requests/http-request.go

```diff
@@ -72,6 +72,11 @@ func (r *HTTPRequest) SetAttackType(attack generators.Type) {
 	r.attackType = attack
 }
 
+// Returns the total number of requests the YAML rule will perform
+func (r *HTTPRequest) GetRequestCount() int64 {
+	return int64( len(r.Raw) | len(r.Path) )
```

Comment:
I can see `raw` requests have precedence over those defined with `path`, should i reflect this logic/behavior in the computation as well? Or do you plan to support both raw and path requests in YAML rules?